build.sbt

```diff
@@ -377,7 +380,8 @@ lazy val spark = (project in file("connectors/spark"))
   .dependsOn(client)
   .settings(
     name := s"$artifactNamePrefix-spark",
-    scalaVersion := "2.12.15",
+    scalaVersion := scala212,
+    crossScalaVersions := Seq(scala212, scala213),
```

Comment:
As per https://www.scala-sbt.org/1.x/docs/Cross-Build.html:

> Note: crossScalaVersions must be set to Nil on the root project to avoid double publishing.